dcs-packaging-tool/dcs-packaging-tool-gui/src/main/java/org/dataconservancy/packaging/gui/presenter/impl/BasePresenterImpl.java

```diff
@@ -46,8 +46,6 @@ protected void bindBaseElements() {
         view.getHeaderViewAboutLink().setOnAction(arg0 -> view.showAboutPopup());
         
         view.getHeaderViewHelpLink().setOnAction(arg0 -> view.showHelpPopup());
-        
-        view.getCancelLink().setOnAction(arg0 -> getController().goToPreviousPage());

```

Comment:
Yeah to allow users to start a new workflow afresh, we clear the PackageMetadata form when user go all the way back to the home screen (where they'd choose which workflow to enter). Essentially restarting the the workflow means a blank PackageState. So we're warning user of the loss of information if they do go all the way back. Wording of the warning could be changed, but that's a minor point/work and can be done once we refine the details.